src/shared/components/DropdownFilter/DropdownItem.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { AutoComplete } from 'antd';
+
+import { TypesIcon } from '../Types/TypesIcon';
+
+const Option = AutoComplete.Option;
+
+export const TypeDropdownItem: React.FunctionComponent<{
+  count: number;
+  key: string;
+  label: string;
+}> = ({ count, key, label }) => {
+  return (
+    // Option does in fact have a label property, the ts warning is a lie.
+    // @ts-ignore (overloaded Options props with label, it works!)
```

Comment:
Are you sure that you don't need to restart you ts-server or something. I'm pretty sure typescript never lies.

Reply:
Looks like typescript is right after all

![Screenshot from 2019-11-01 15-35-06](https://user-images.githubusercontent.com/4364154/68032089-3afa2000-fcbd-11e9-95a4-ce813369e08b.png)